server_addon/houdini/client/ayon_houdini/startup/otls/ayon_lop_import.hda/ayon_8_8Lop_1lop__import_8_81.0/DialogScript

```diff
@@ -0,0 +1,326 @@
+# Dialog script for ayon::lop_import::1.0 automatically generated
+
+{
+    name	ayon::lop_import::1.0
+    script	load_asset::1.0
+    label	"Load Asset"
+
+    help {
+	""
+    }
+
+    inputlabel	1	"Input Stage"
+    inputlabel	2	"Input 2"
+    inputlabel	3	"Input 3"
+    inputlabel	4	"Input 4"
+
+    groupsimple {
+        name    "info2"
+        label   "Info"
+        parmtag { "script_callback" "hou.phm().refresh_available_versions(kwargs['node'])" }
+        parmtag { "script_callback_language" "python" }
+
+        parm {
+            name    "assetinfo_labelparm"
+            label   "Heading"
+            type    label
+            default { "Choose Product" }
+            parmtag { "sidefx::look" "heading" }
+        }
+        parm {
+            name    "project_name"
+            label   "Project"
+            type    string
+            default { "$AYON_PROJECT_NAME" }
```

Comment:
This adds the same select button next to proejcts
```suggestion
            default { "$AYON_PROJECT_NAME" }
            parmtag { "script_action" "from ayon_houdini.api.hda_utils import select_folder_path;select_folder_path(kwargs['node'])" }
            parmtag { "script_action_icon" "BUTTONS_reselect" }
```

I was concerned what if user clicked the selection button next project and and the one next to folder path. 
and it was funny that Houdini prevented that from happening because selection button forces the user to make a selection or close the window.


![Animation_79](https://github.com/ynput/ayon-core/assets/20871534/4a2c9bc8-a01f-4489-b2e2-37d190a2cf4c)

---

Also, I loved how Houdini propagated the changes live, once I modified the file it changed the definition while I was using the HDA